apis/catalog/v1alpha1/elasticsearch_version_types.go

```diff
@@ -57,7 +57,7 @@ type ElasticsearchVersionSpec struct {
 	// Exporter Image
 	Exporter ElasticsearchVersionExporter `json:"exporter" protobuf:"bytes,4,opt,name=exporter"`
 	// Tools Image
```

Comment:
Add `// +optional` comment and then run `make gen fmt`

Reply:
Added